unlock-app/src/__tests__/components/Unlock.test.js

```diff
@@ -1,57 +0,0 @@
-import React from 'react'
-import { shallow } from 'enzyme'
-import { Unlock } from '../../components/Unlock'
-
-let config, network
-
-describe('Unlock Component', () => {
-
-  beforeEach(() => {
-    config = {
-      providers: {},
-      isRequiredNetwork: () => true,
-      requiredNetwork: 'dev',
-    }
-    network = {
-      name: 4,
-    }
-  })
-
-  describe('when there are no providers', () => {
```

Comment:
These are important tests. Are they being moved somewhere else?

Reply:
You are right, I will move them these under testing `withConfig`.